portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,17 +48,16 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println(request.getParameter("lang"));
-
     List<String> comments = new ArrayList<>();
     Query query = new Query(ENTITY_KIND).addSort(ENTITY_TIMESTAMP_HEADER, SortDirection.ASCENDING);
     PreparedQuery results = datastore.prepare(query);
-
+    String prevLanguageCode = request.getParameter("from");
```

Comment:
"from" and "to" are too common words. Can you use maybe source_language and target_language?

And in order to be consistent, I think you can use sourceLanguageCode and targetLanguageCode.

Reply:
Done